test/getusers.test.js

```diff
@@ -0,0 +1,48 @@
+const request = require('supertest');
+const app = require('../app');
+
+const mockUser = {
+  name: 'Richard',
+  last_name: 'Feynman',
+  email: 'r.feynman@wolox.co',
+  password: '2w1321AScsda#'
+};
+
+const mockCredentials = {
+  email: 'r.feynman@wolox.co',
+  password: '2w1321AScsda#'
+};
+
+describe('GET /users', () => {
+  let mockToken = null;
+  beforeEach(async () => {
+    await request(app)
+      .post('/users')
+      .send(mockUser);
```

Comment:
To void this, you can use factories. You can create what you need in the db

Reply:
Fixed it